<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monolayer</class>
 <widget class="QMainWindow" name="Monolayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>585</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>790</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Monolayer_2">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Trilayer Sheet</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>222</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Number of kpoints</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="Nk">
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>150</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>t</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="Hoppingt2">
       <property name="whatsThis">
        <string extracomment="INtra layer hopping"/>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>3.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>t'</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="Hoppingtprime2">
       <property name="whatsThis">
        <string extracomment="Inter layer hopping"/>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Magnetic Field Type</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="SelectBBil"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="Magneticfield2">
       <property name="toolTipDuration">
        <number>-2</number>
       </property>
       <property name="whatsThis">
        <string extracomment="Magnetic field strength"/>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="minimum">
        <double>-5.000000000000000</double>
       </property>
       <property name="maximum">
        <double>5.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Angle</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="BAngle">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="maximum">
        <double>4.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Sublattice Imbalance</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDoubleSpinBox" name="OnsiteV_2">
       <property name="whatsThis">
        <string extracomment="On-site potential"/>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Stacking Type</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QComboBox" name="SelectStack"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>135</width>
      <height>128</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Axis kx</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="xIndex">
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>17</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Axis dkx</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="dxIndex">
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>75</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="dyIndex">
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>75</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="yIndex">
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>75</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Axis ky</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Axis dky</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="PlotSummary">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Bandstructure</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Topology</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Berry">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Berry phase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LocChernKprime">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Local Chern number K'</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Chern">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Chern number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LocChernK">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Local Chern number K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Phases">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Phases</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DOS">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>480</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>DOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of k points</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Energy window</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numberofKop">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Operators</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="energy">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>500</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-2</number>
    </property>
    <property name="whatsThis">
     <string extracomment="Magnetic field strength"/>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedKingdom"/>
    </property>
    <property name="minimum">
     <double>-5.000000000000000</double>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMaster_Project">
    <property name="title">
     <string>Master Project</string>
    </property>
    <addaction name="actionMonolayer"/>
   </widget>
   <addaction name="menuMaster_Project"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMonolayer">
   <property name="text">
    <string>Bilayer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
